docs/ref/migration-operations.txt

```diff
@@ -413,7 +413,7 @@ if ``atomic=True`` is passed to the ``RunPython`` operation.
 A highly specialized operation that let you mix and match the database
 (schema-changing) and state (autodetector-powering) aspects of operations.
 
-It accepts two list of operations, and when asked to apply state will use the
+It accepts two lists of operations, and when asked to apply state will use the
```

Comment:
I would also chop `"It "` to be consistent with other methods documentation.

Reply:
I think it reads a bit better with the "It" because unlike the other operations, this isn't the first paragraph.